### PR TITLE
support New TileSet Editor zooming use CTRL/CMD + MouseWheel

### PR DESCRIPTION
**Pull request by Liangdi**
issue #22976 , i have implement the zooming action use CTRL/CMD + MouseWheel

### CODE REVIEW

**Comment by Zylann** [comment]
Why is CTRL required? The viewport and texture region editor lets you zoom in/out with just mouse wheel.

**Comment by Liangdi** [comment]
> Why is CTRL required? The viewport and texture region editor lets you zoom in/out with just mouse wheel.

In my opinion , some operations such as drawing a polygon sometime need to scroll the view, the default mousewheel handle is to scroll the view, so add CTRL when zoom in/out

File: editor/plugins/tile_set_editor_plugin.cpp [hunk 1]
```diff
@@ -972,6 +977,18 @@ void TileSetEditor::_on_workspace_input(const Ref<InputEvent> &p_ie) {
...
+		// Mouse Wheel Event
+		const int _mouse_button_index = mb->get_button_index();
+		if (_mouse_button_index == BUTTON_WHEEL_UP && mb->get_control()) {
+			// Zoom In
+			_zoom_in();
+
+		} else if (_mouse_button_index == BUTTON_WHEEL_DOWN && mb->get_control()) {
+			// Zoom Out
+			_zoom_out();
+			//KEY_MASK_CMD
```
Comment (akien-mga) [inline editor/plugins/tile_set_editor_plugin.cpp]: Please remove this comment.

**Comment by Liangdi** [comment]
> Please squash the commits together, see [PR workflow](http://docs.godotengine.org/en/latest/community/contributing/pr_workflow.html#mastering-the-pr-workflow-the-rebase).

thanks , i have squashed the commits

**Comment by akien-mga** [comment]
Thanks!

**Pull request merged**